### ISSUE TITLE
Does this project work?

### ISSUE THREAD

**Issue by louisabraham** [johang88/haskellinjavascript#3]
I cloned the repository, and opened some html files but have a lot of errors.

**Comment by Untitled-png** [comment]
yeah... it's weird how it has so many stars yet no readme or any information lol
